### ISSUE TITLE
Uncaught TypeError: Cannot read property 'relativePath' of undefined

### ISSUE THREAD

**Issue by Hidove** [kartik-v/bootstrap-fileinput#1481]
![image](https://user-images.githubusercontent.com/42663943/66330299-b2f55480-e962-11e9-8e60-9b730e7317c5.png)
![image](https://user-images.githubusercontent.com/42663943/66330323-bee11680-e962-11e9-93c4-bb7910365f3d.png)

**Comment by kartik-v** [comment]
You are passing a custom blob without any relativeFilePath unlike a file selected via native HTML file input. In that case try adding a `id` as an additional parameter to `addToStack` in addition to the file blob which should be an unique identifier for the file - see the [`addToStack` method documentation](http://plugins.krajee.com/file-input/plugin-methods#addToStack)

**Comment by Hidove** [comment]
为什么我用4.5.1版本的可以呢？

**Comment by Hidove** [comment]
I use the index of the loop as id, but the result is still not good. I use getFileStack to pass in the file group but upload it.
![image](https://user-images.githubusercontent.com/42663943/66720269-2ef11000-ee2d-11e9-8bd5-95ac84b3ec2a.png)
![image](https://user-images.githubusercontent.com/42663943/66720277-4e883880-ee2d-11e9-824c-dd1a6b59dbe4.png)

**Comment by Hidove** [comment]
![image](https://user-images.githubusercontent.com/42663943/66720380-6ad8a500-ee2e-11e9-9b52-024e47884638.png)
I used getFileStack to pass in the file group, and then I had to manually click the upload button to upload it myself.
![image](https://user-images.githubusercontent.com/42663943/66720403-9fe4f780-ee2e-11e9-8fc4-0158186fce19.png)
![image](https://user-images.githubusercontent.com/42663943/66720424-1255d780-ee2f-11e9-86bb-3a5c83698a8d.png)
Similarly, I used addToStack to upload the file, and then I manually clicked the upload button to upload it. I also made a mistake. I checked the getFileList and found that the content was the same, but the upload failed or failed.
